lib/ansible/modules/packaging/os/zypper_repository.py

```diff
@@ -88,7 +87,7 @@
     auto_import_keys:
         description:
             - Automatically import the gpg signing key of the new or changed repository.
-            - Has an effect only if state is I(present). Has no effect on existing (unchanged) repositories or in combination with I(absent).
+            - Has an effect only if I(state=present). Has no effect on existing (unchanged) repositories or in combination with I(state=absent).
```

Comment:
I just tried to follow [http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html](URL) ... Formatting options
and I thought
```I(state=present)``` is better to understand than ```state is I(present)```